Assets/Scripts/UI/UIHighFrequencyPeer2PeerMenu.cs

```diff
@@ -0,0 +1,369 @@
+            FriendsManager = EOSManager.Instance.GetOrCreateManager<EOSFriendsManager>();
+            Peer2PeerManager.ParticleController = ParticleManager;
+            Peer2PeerManager.owner = this;
+            Peer2PeerManager.parent = this.transform;
+        }
+
+        private void OnDestroy()
+        {
+            EOSManager.Instance.RemoveManager<EOSHighFrequencyPeer2PeerManager>();
+            EOSManager.Instance.RemoveManager<EOSFriendsManager>();
+        }
+        public void ChatMessageEndEdit(string arg0)
+        {
+            SendOnClick();
+        }
+
+        public void ToggleHighFrequencySending()
+        {
+            Peer2PeerManager.sendActive = !Peer2PeerManager.sendActive;
+        }
+        private void Update()
+        {
+            ProductUserId messageFromPlayer = Peer2PeerManager.HandleReceivedMessages();
+            if (messageFromPlayer != null)
+            {
+                IncomingChat(messageFromPlayer);
+            }
+
+            if (currentChatProductUserId == null || !currentChatProductUserId.IsValid())
+            {
+                return;
+            }
+
+            /*switch (Peer2PeerManager.GetNATType())
+            {
+                case NATType.Moderate:
+                    NATTypeText.text = "Moderate";
+                    break;
+                case NATType.Open:
+                    NATTypeText.text = "Open";
+                    break;
+                case NATType.Strict:
+                    NATTypeText.text = "Strict";
+                    break;
+                case NATType.Unknown:
+                    NATTypeText.text = "Unknown";
+                    break;
+            }*/
+
+           /*if (Peer2PeerManager.GetChatDataCache(out Dictionary<ProductUserId, ChatWithFriendData> ChatDataDictionary))
+            {
+                // Destroy current UI chat entry list
+                foreach (Transform child in ChatEntriesContentParent.transform)
+                {
+                    GameObject.Destroy(child.gameObject);
+                }
+
+                if (ChatDataDictionary.TryGetValue(currentChatProductUserId, out ChatWithFriendData chatData))
+                {
+                    foreach (ChatEntry entry in chatData.ChatLines)
+                    {
+                        GameObject chatEntryUIObj = Instantiate(ChatEntryPrefab, ChatEntriesContentParent.transform);
+                        UIChatEntry uiEntry = chatEntryUIObj.GetComponent<UIChatEntry>();
+
+                        if (entry.isOwnEntry)
+                        {
+                            uiEntry.SetLocalEntry("Me", entry.Message);
+                        }
+                        else
+                        {
+                            uiEntry.SetRemoteEntry(currentChatDisplayName, entry.Message);
+                        }
+                    }
+                }
+            }*/
```

Comment:
For the sake of clarity - is there any reason we cannot get rid of this code? If the concern is having it around for historical context, I would argue that is what version control is for.